source/endpoints/jobs.rst

```diff
@@ -250,5 +250,66 @@ Code                   Description
                           "type": "https://www.rev.ai/api/v1/errors/out-of-credit",
                           "detail": "You have only 60 seconds remaining",
                           "current_balance": 60
-                        }    
+                        }
+====================== ===============================================================
+
+``DELETE /jobs/{id}``
+*****************
+
+Deletes a transcription job by id.
```

Comment:
I would add something like "All data related to the job, such as input media and transcript, will be permanently deleted. A job can only by deleted once it's completed (either with success or failure)."

Reply:
I'll add it to the implementation PR for the OpenApi docs